editors/code/src/workspace.ts

```diff
@@ -0,0 +1,51 @@
+import path from "path";
+import * as vscode from "vscode";
+import * as fs from "fs-extra";
+
+export async function getRootWorkspaceFolder(): Promise<vscode.WorkspaceFolder> {
```

Comment:
This also comes from https://github.com/microsoft/vscode-python-tools-extension-template/blob/8f474ec4ac4e7205ffed9f7f243473bb00bf29c0/src/common/utilities.ts#L38

It seems reasonable?

Reply:
For some definition of reasonable ;)
I mean I would expect all this complexity to be hidden behind a VS Code API.